autonomous.py

```diff
@@ -111,8 +113,37 @@ def executeAuton(self):
             inaccuracyDistance = math.sqrt(pow(relativePose.X(), 2) + pow(relativePose.Y(), 2))
             PPLibTelemetry.setPathInaccuracy(inaccuracyDistance)
 
-            #if the speeds are minimal and minimum time has elapsed, move onto the next task
-            if(abs(self.chassisSpeeds.vx/self.maxSpeed) < 0.1 and abs(self.chassisSpeeds.vy/self.maxSpeed) < 0.1 and abs(self.chassisSpeeds.omega_dps) < 5 and self.autonTimer.get() - self.lastTime > self.pathTrajectory.getTotalTimeSeconds()):
+            self.noteDriveCounter += 1
+
+            #if there is a note, it is within range, waitTime isn't negative, and the waitTime has passed, then use note detection
+            if(self.notedetector.hasTarget() and waitTime >= 0 and self.notedetector.getTargetErrorY() < self.maxPickUpDistance and self.autonTimer.get() - self.lastTime > waitTime):
+                #move every other robot cycle
+                if(self.noteDriveCounter % 2 == 0):
```

Comment:
You may want to pull the `2` into a variable to make it easier to tune over time.